src/core/grpc_service.proto

```diff
@@ -264,26 +272,17 @@ message SharedMemoryControlRequest
     //@@
     message CUDASharedMemoryIdentifier
     {
-      //@@  .. cpp:var:: string shared_memory_key
-      //@@
-      //@@     The name of the system shared memory region that holds the
-      //@@     cudaIPC handle.
+      //@@  .. cpp:var:: bytes raw_handle (repeated)
       //@@
-      string shared_memory_key = 1;
-
-      //@@  .. cpp:var:: uint64 offset
-      //@@
-      //@@     The offset of the cudaIPC handle from the start of the shared
-      //@@     memory region.
-      //@@     start = offset, end = offset + size;
+      //@@     The raw serialized cudaIPC handle.
       //@@
-      uint64 offset = 2;
+      repeated bytes raw_handle = 1;
```

Comment:
repeated bytes is multiple "arrays of bytes". You just want "bytes".

Reply:
Fixed